cvat-ui/src/components/labels-editor/label-form.tsx

```diff
@@ -374,7 +374,6 @@ export default class LabelForm extends React.Component<Props> {
     private renderLabelNameInput(): JSX.Element {
         const { label, labelNames } = this.props;
         const value = label ? label.name : '';
```

Comment:
Is `const value = label?.name || ''` will works  here instead?

Reply:
Guess it's quite the same thing